docs/workflow-messaging.md

```diff
@@ -0,0 +1,77 @@
+## Workflow queues and topics - questions/decisions
+
+There are two SNS topics owned by storage:
+
+- born-digital-bag-notifications-prod
+- born-digital-bag-notifications-staging
+
+There will be two new SNS topics for Goobi:
+
+- digitised-bag-notifications-prod
+- digitised-bag-notifications-staging
+
+_(I assume, or similar names anyway)_
+
+There are/will be queues subscribed to these topics. Where do the queues live? We have currently one production queue, known as:
```

Comment:
> Where do the queues live?

I'd prefer they live in digirati acct. Keeps all of the subscriber logic in one place and the publishing logic in another.